### ISSUE TITLE
Research solutions to Netlify js bundle cache invalidation

### ISSUE THREAD

**Issue by mateoclarke** [cityofaustin/atd-mobile-signals-work-orders#163]
- https://www.netlify.com/blog/2017/02/23/better-living-through-caching/

**Comment by mateoclarke** [comment]
@sergiogcx, did you see this article by Netlify? It seems like they have some defaults and opinions around caching: https://www.netlify.com/blog/2017/02/23/better-living-through-caching/

And the suggest a specific way to overrider their headers:
https://www.netlify.com/docs/headers-and-basic-auth/#custom-headers

**Comment by sergiogcx** [comment]
@mateoclarke I would say let's test the max-age to 0 (no-cache) and see how bad performance would be, then try with:

- One hour: max-age=3600
- One day: max-age=86400
- One week: max-age=604800

**Comment by sergiogcx** [comment]
Changed milestone to 7, this is going to take more time to test and monitor.

**Comment by sergiogcx** [comment]
PR:
https://github.com/cityofaustin/atd-mobile-signals-work-orders/pull/205


Got it to work, first I tested with a header of 60 seconds, to see if the cache would actually work.

1. Visited the site for priming the cache in Chrome.
2. Reloaded the site immediately, checked the network console to see if loaded from memory:
![Screen Shot 2019-09-23 at 4.08.10 PM.png](https://images.zenhubusercontent.com/5b7edad7290aac725aec290c/3869e3e9-edc4-4979-b305-29cb63c5a913)
3. Checked two minutes later, then compared results:
![Screen Shot 2019-09-23 at 4.10.58 PM.png](https://images.zenhubusercontent.com/5b7edad7290aac725aec290c/214883df-53bd-4324-8a1b-a7cb6343ec13)
